README.md

```diff
@@ -1,2 +1,47 @@
-# readme-generator-cli
-Readme generator using inquirer CLI
+# Professional README Generator
+
+A command-line application that dynamically generates a professional README.md file from a user's input using the [Inquirer package](https://www.npmjs.com/package/inquirer).
+
+The application is invoked by using the following command:
+
+```bash
+node index.js
+```
+
+## User Story
+
+```md
+AS A developer
+I WANT a README generator
+SO THAT I can quickly create a professional README for a new project
+```
+
+```md
+GIVEN a command-line application that accepts user input
+WHEN I am prompted for information about my application repository
+THEN a high-quality, professional README.md is generated with the title of my project and sections entitled Description, Table of Contents, Installation, Usage, License, Contributing, Tests, and Questions
+WHEN I enter my project title
+THEN this is displayed as the title of the README
+WHEN I enter a description, installation instructions, usage information, contribution guidelines, and test instructions
+THEN this information is added to the sections of the README entitled Description, Installation, Usage, Contributing, and Tests
+WHEN I choose a license for my application from a list of options
+THEN a badge for that license is added near the top of the README and a notice is added to the section of the README entitled License that explains which license the application is covered under
+WHEN I enter my GitHub username
+THEN this is added to the section of the README entitled Questions, with a link to my GitHub profile
+WHEN I enter my email address
+THEN this is added to the section of the README entitled Questions, with instructions on how to reach me with additional questions
+WHEN I click on the links in the Table of Contents
+THEN I am taken to the corresponding section of the README
+```
+
+## Video
+
+A walkthrough video demonstrating the functionality of the application is available [here](https://drive.google.com/file/d/18uOOK6Qo_rYurWdfsqoAXSIAzDgvkzQf/view?usp=sharing)
+
+## README.md sample
+
+A sample README.md file for a project repository generated using the application is available [here](https://drive.google.com/file/d/1WY07SjwmVK1W53OMAH0l1_E_TeQ1sfH3/view?usp=sharing)
+
+## Deployed URL
```

Comment:
This application isn't deployed, so it's a bit confusing to have this section in the readme.